### PR TITLE
Add LinearOperators 2.2 to compat

### PR DESCRIPTION
**Pull request by geoffroyleconte**

### CODE REVIEW

**Comment by geoffroyleconte** [comment]
I think that the breakage failures are because of compatibility issues that can be fixed if the NLPModels and NLPModelsModifiers dependancies of the other packages are updated.
However I do not know why the FreeBSD tests fail, does anybody know?

**Comment by tmigot** [comment]
> I think that the breakage failures are because of compatibility issues that can be fixed if the NLPModels and NLPModelsModifiers dependancies of the other packages are updated. However I do not know why the FreeBSD tests fail, does anybody know?

Just to confirm that the breaking packages are not yet fully updated to NLPModels 0.18 ;).
For the other issue, I don't know...

**Comment by tmigot** [comment]
This error has been here for a while actually, since version 0.4.0 (or since the change master to main), and we have been ignoring it...

File: Project.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
 FastClosures = "0.2.1, 0.3.0"
-LinearOperators = "2.0"
+LinearOperators = "2.0, 2.2"
```
Comment (tmigot) [inline Project.toml:15]: Is it suppose to be working with `2.1` also? If so, could we say `LinearOperators = "2"` instead of listing the releases?

**Comment by dpo** [comment]
https://github.com/cirruslabs/cirrus-ci-agent/issues/201

**Comment by dpo** [comment]
FreeBSD fixed! 🎉

**Comment by geoffroyleconte** [comment]
Nice thanks! Do I need to re-run all tests?

**Comment by dpo** [comment]
No, this should be good to go

**Review by tmigot — APPROVED** [review]

**Pull request merged**